contracts/platform/bondage/Bondage.sol

```diff
@@ -252,13 +252,14 @@ contract Bondage is Destructible, BondageInterface, Upgradable {
     }
 
     /**** Get Methods ****/
-    function isProviderInitialized(address holderAddress, address oracleAddress) public view returns (bool) {
+    function isProviderInitialized(address holderAddress, address oracleAddress) public payable returns (bool) {
         return db.getNumber(keccak256(abi.encodePacked('holders', holderAddress, 'initialized', oracleAddress))) == 1 ? true : false;
     }
 
     /// @dev get broker address for endpoint
-    function getEndpointBroker(address oracleAddress, bytes32 endpoint) public view returns (address) {
-        return address(db.getBytes32(keccak256(abi.encodePacked('oracles', oracleAddress, endpoint, 'broker'))));
+    function getEndpointBroker(address oracleAddress, bytes32 _endpoint) public payable returns (address) {
+        db.setBytes32(keccak256(abi.encodePacked('oracles', oracleAddress, _endpoint, 'broker')), _endpoint);
```

Comment:
compiler was asking for 2 inputs when it only got 1.

changes i put was _endpoint for the second parameter and i let db.setbyte32 first before casting and returning. instead of returning in one time